distributed/asyncio.py

```diff
@@ -0,0 +1,88 @@
+# flake8: noqa
+
+import asyncio
+from functools import wraps
+
+from tornado.gen import is_coroutine_function
+from tornado.platform.asyncio import BaseAsyncIOLoop
+from tornado.platform.asyncio import to_asyncio_future, to_tornado_future
+
+from .client import Client, Future
+
+from tornado.ioloop import IOLoop
+
+
+def to_asyncio(method):
+    @wraps(method)
+    def convert(*args, **kwargs):
+        return to_asyncio_future(method(*args, **kwargs))
+    return convert
+
+
+class AioLoop(BaseAsyncIOLoop):
+
+    @property
+    def _running(self):
+        """Distributed checks IOLoop's _running property extensively"""
+        return self.asyncio_loop.is_running()
+
+
+class AioFuture(Future):
+
+    def __await__(self):
+        return self.result().__await__()
+
+    result = to_asyncio(Future._result)
+    exception = to_asyncio(Future._exception)
+    traceback = to_asyncio(Future._traceback)
+
+
+class AioClient(Client):
+
```

Comment:
This could use a small docstring.  We should probably also include it at least in the API documentation.  Although if you think that this is important then we should probably include a small docpage.